Cargo.toml

```diff
@@ -52,3 +52,6 @@ chrono = ["chrono-time", "diesel/chrono"]
 r2d2 = ["diesel/r2d2"]
 dynamic-schema = ["diesel-dynamic-schema"]
 gst = []
+
+[patch.crates-io]
```

Comment:
Of curiosity: I take it the patch is to use some unofficial pre-release version of diesel 2.1 with the latest trait changes and fixes? Or is there something more?

Reply:
* That diesel patch does only provide some additional ways to configure that specific syntax, the real fix happens in diesel-oci by using this flag + generating the right sql syntax. That happens mostly here: https://github.com/GiGainfosystems/diesel-oci/pull/51/files#diff-206a297303e4c39e996fa27af96fdcfd7d7075a4e42fc4597afe1cfbad835004R58-R70
* This patch is already merged upstream: https://github.com/diesel-rs/diesel/pull/3722, so it's not unoffical anymore. It's just not released, as this is strictly a new feature so it will be in diesel 2.2